examples/vertx-example/layrry-links-tournament/src/main/java/module-info.java

```diff
@@ -1,8 +1,5 @@
-import org.moditect.layrry.example.links.core.spi.RouterContributor;
```

Comment:
Any particular reason for changing this?

Reply:
The license plugin would wipe out the imports and file header for no reason other than having those imports at the top of the file